### ISSUE TITLE
(A11y severity 4) Convert anchor tags to buttons where appropriate

### ISSUE THREAD

**Issue by sathomas** [oaeproject/3akai-ux#3962]
Anchor tags that are used as buttons (e.g. `<a class="as-close">` for auto-suggest items) should be converted to `<button>` tags where possible to improve accessibility.
